front/app/utils/helperUtils.ts

```diff
@@ -1,3 +1,4 @@
+import { uuid4 } from '@sentry/utils';
```

Comment:
Is there a way to not use this library for this. It's the only import we use for it, Sentry has a completely different purpose and I think we should delete this dependency.

Reply:
Sure, I just copied what was already there

Reply:
Yes, not sure why we did this in the first place. This was on my radar from before. Would be great to get rid of it 🙌